build/Targets/References.targets

```diff
@@ -222,7 +222,7 @@
       <ItemGroup>
 
         <PackageReference Include="Moq" Version="4.2.1402.2112" />
-        <PackageReference Include="xunit" Version="2.1.0" />
+        <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
```

Comment:
Jon's moved to 2.2.0 in the latest commit. The perf looks fine - lets go ahead with this.